kolibri/core/errorreports/models.py

```diff
@@ -5,7 +5,15 @@
 from django.utils import timezone
 
 from .constants import POSSIBLE_ERRORS
+from .schemas import context_backend_schema
+from .schemas import context_frontend_schema
+from .schemas import default_context_backend_schema
+from .schemas import default_context_frontend_schema
+from kolibri import VERSION
```

Comment:
I would much prefer we use the full unqualified version here: https://github.com/learningequality/kolibri/blob/develop/kolibri/__init__.py#L18